### PR TITLE
HDDS-7505. Recon: Rename OM DB Sync to DB SYNC on UI

### PR DESCRIPTION
**Pull request by smitajoshi12**
## What changes were proposed in this pull request?
Change label from OM DB to DB Sync on UI of Recon.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-7505

## How was this patch tested?
- Manually

image

### CODE REVIEW

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ class AutoReloadPanel extends React.Component<IAutoReloadPanelProps> {
      const lastUpdatedDeltaFullText = lastUpdatedOMDBDelta === 0 || lastUpdatedOMDBDelta === undefined || lastUpdatedOMDBFull === 0 || lastUpdatedOMDBFull === undefined ? '' :
      (
       <>
-      &nbsp; | OM DB updated at {lastUpdatedDeltaFullToolTip}
+      &nbsp; | DB Sync updated at {lastUpdatedDeltaFullToolTip}
```
Comment (aswinshakil) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx:80]:
```suggestion
      &nbsp; | DB Synced at {lastUpdatedDeltaFullToolTip}
```
`DB Sync updated` sounds different.

Comment (smitajoshi12) [reply hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx]: Completed Changes Please review.

**Comment by kerneltime** [comment]
@dombizita

**Comment by kerneltime** [comment]
@GeorgeJahad

**Comment by GeorgeJahad** [comment]
lgtm

**Review by aswinshakil — APPROVED** [review]

**Comment by dombizita** [comment]
A discussion started at #3929, which lead us to this jira and patch. What do you think of this approach instead of `Cluster sync`? Do you have any other ideas @myskov? 
cc @smengcl

**Comment by devmadhuu** [comment]
ClusterSync terminology is better as we have few other tasks and sub-tasks also which do  processing on data synced from OM and SCM DB both on a periodic basis and few internal tasks for SCM container cache sync (#3947 )

**Comment by dombizita** [comment]
> ClusterSync terminology is better as we have few other tasks and sub-tasks also which do processing on data synced from OM and SCM DB both on a periodic basis and few internal tasks for SCM container cache sync (#3947 )

because of that I think we should keep `OM sync`, as we kind of have `SCM sync` and that is not happening with this button. also the `last refreshed` is not correct regarding SCM, only the OM DB related data is refreshed. I still don't think this rename is useful, as users will believe all the data related to the cluster is refreshed at a time and they can do that with the button, but that is not true. it is better to have something that users can look up to and search for its meaning, than misleading them.

one way is to change it to `DB sync` and later we can add one for SCM too as it is advised [here](https://github.com/apache/ozone/pull/3929#issuecomment-1314287362).

@myskov any thoughts on this?

**Comment by DaveTeng0** [comment]
hey @myskov ~ there are some questions above waiting your feedback. Please help take a look! Thanks!

**Comment by DaveTeng0** [comment]
should we hold this change for now~?

**Comment by smitajoshi12** [comment]
We can merge PR as it is only name change.

**Pull request merged**